pkg/build/make.go

```diff
@@ -18,101 +18,30 @@ package build
 
 import (
 	"fmt"
-	"go/build"
-	"os"
-	"path/filepath"
 
-	"k8s.io/klog"
 	"sigs.k8s.io/kubetest2/pkg/exec"
 )
 
-type MakeBuilder struct{}
+type MakeBuilder struct {
+	RepoRoot string
+}
 
 var _ Builder = &MakeBuilder{}
 
 const (
 	target = "quick-release"
 )
 
-var (
-	// This will need changed to support other platforms.
-	tarballsToExtract = []string{
-		"kubernetes.tar.gz",
-		"kubernetes-test-linux-amd64.tar.gz",
-		"kubernetes-test-portable.tar.gz",
-		"kubernetes-client-linux-amd64.tar.gz",
-	}
-)
-
-// Build builds kubernetes with the bazel-release make target
+// Build builds kubernetes with the quick-release make target
 func (m *MakeBuilder) Build() (string, error) {
-	// TODO(RonWeber): This needs options
-	src, err := K8sDir("kubernetes")
-	if err != nil {
-		return "", err
-	}
-	version, err := sourceVersion(src)
+	version, err := sourceVersion(m.RepoRoot)
 	if err != nil {
 		return "", fmt.Errorf("failed to get version: %v", err)
 	}
-	cmd := exec.Command("make", "-C", src, target)
+	cmd := exec.Command("make", "-C", m.RepoRoot, target)
```

Comment:
Just checking, the Make file exists at Root?

Reply:
yup, https://github.com/kubernetes/kubernetes/blob/master/Makefile

Reply:
it's actually a symlink, but yes 🙃 
the real file is in ./build/root/Makefile